camera/src/definitions.ts

```diff
@@ -0,0 +1,222 @@
+export type CameraPermissionState = PermissionState | 'limited';
```

Comment:
Where is `PermissionState` imported?

Reply:
Does it need to be imported? It's defined in `lib.dom.d.ts`.

Reply:
Yeah, it should use our definition from `@capacitor/core` which includes `prompt-with-rationale`. `lib.dom.d.ts` are global TypeScript types for the browser DOM, not our types.